gaia-fast-list.js

```diff
@@ -637,6 +646,7 @@ Internal.prototype = {
       }
 
       hash[section].push(item);
+      this.hasSections = true;
```

Comment:
This is a micro-optimization to replace the slightly more expensive `.hasSections()` method.